exasol/analytics/query_handler/result.py

```diff
@@ -26,3 +27,4 @@ class Continue(Result):
 @dataclass()
 class Finish(Generic[T], Result):
     result: T
+    audit_data: AuditData | None = None
```

Comment:
I think, we don't want to allow a write query here

```suggestion
    audit_data: AuditQuery | None = None
```

Reply:
`AuditData` is not a _query_ (neither write nor read), but just "data". 
Not sure how this relates to "write" or "read"?